### DISCUSSION TITLE
Seitenfarben

### DISCUSSION THREAD

**Discussion by tschortschde** [mcuiobroker/docuJarvis#24] [Q&A]
Hallo,
ich nutze die Version v3.1.2 pro und möchte gerne die Farben der Seiten usw. anpassen.
Leider finde ich die entsprechende Einstellung nicht....
Gemäß Anleitung sollte es so aussehen:
![Screenshot 2023-03-24 124632](https://user-images.githubusercontent.com/11274233/227513344-fcb5da87-f7be-4c3a-88d9-88aae27bf5bc.jpg)

Bei mir sieht es aber so aus:
![Screenshot 2023-03-24 124921](https://user-images.githubusercontent.com/11274233/227513715-7322e163-facb-4f9d-ad20-0200b557382e.jpg)
Kannst Du mir bitte weiterhelfen?
VG

**Comment by mcuiobroker** [comment]
Hintergrund wird über den Body Stil definiert. 
`{"background-color:"red"}`

**Reply by tschortschde** [reply]
Und die Schriftfarbe?

**Comment by mcuiobroker** [comment]
{"color":"blue","background-color":"red"}
